cmd/acra-translator/http_api/decryptor.go

```diff
@@ -106,7 +107,23 @@ func (decryptor *HTTPConnectionsDecryptor) ParseRequestPrepareResponse(logger *l
 		if err != nil {
 			msg := fmt.Sprintf("Can't decrypt AcraStruct")
 			logger.WithError(err).WithField(logging.FieldKeyEventCode, logging.EventCodeErrorTranslatorCantDecryptAcraStruct).Warningln(msg)
-			return responseWithMessage(request, http.StatusUnprocessableEntity, msg)
+			response := responseWithMessage(request, http.StatusUnprocessableEntity, msg)
+			// check poison records
+			poisoned, err := base.CheckPoisonRecord(acraStruct, decryptor.keystorage)
+			if err != nil {
+				logger.WithError(err).Errorln("Can't check is it poison record")
```

Comment:
I would re-phrase log to
"Can't check for poison record, possible missing Poison record decryption key"